driver-core/src/main/java/com/datastax/driver/core/Connection.java

```diff
@@ -669,7 +623,7 @@ public long getReadTimeoutMillis() {
         private Bootstrap newBootstrap() {
             Bootstrap b = new Bootstrap();
             b.group(eventLoopGroup)
-                .channel(CHANNEL_CLASS);
+                .channel(channelClass);
```

Comment:
Couldn't you just do nettyOptions.channelClass() here instead of having a separate instance variable?